configure.ac

```diff
@@ -145,6 +146,20 @@ CPPFLAGS="-D_REENTRANT -D_THREAD_SAFE $CPPFLAGS"
 # For host type checks
 AC_CANONICAL_HOST
 
+#For a working getsockopt() optname=SO_ERROR
+dnl AC_MSG_CHECKING([for getsockopt optname SO_ERROR)])
```

Comment:
Luca Boccassi, on Sat 16 Jan 2016 05:17:58 -0800, wrote:
> > +#For a working getsockopt() optname=SO_ERROR
> > +dnl AC_MSG_CHECKING([for getsockopt optname SO_ERROR)])
> 
> On the Debian bug discussion [1], [2]@sthibaul mentioned that this sockopt is
> in fact available on Hurd. Also doesn't look like HAVE_SOCKLEN_T is used. Any
> reason to keep this section?

I don't see why this is there either. Svante?

Samuel